tc/core/polyhedral/cuda/tighten_launch_bounds.cc

```diff
@@ -75,6 +75,50 @@ inline std::vector<const detail::ScheduleTree*> leaves(
       [](const detail::ScheduleTree* st) { return st->numChildren() == 0; },
       detail::ScheduleTree::collect(tree));
 }
+
+/*
+ * Compute the maximal value attained by the mapping parameter "id".
+ * Return std::numeric_limits<size_t>::max() if this value cannot
+ * be determined.
+ */
+size_t max(const Scop& scop, const mapping::MappingId& id) {
```

Comment:
Can we have a longer name here?   It's easy to confuse with `std::max` given that we are often `using namespace std`.